src/native_module/core.cljs

```diff
@@ -609,3 +609,15 @@
             {:fn          :get-connection-string-for-exporting-keypairs-keystores
              :config-json config-json})
   (.getConnectionStringForExportingKeypairsKeystores ^js (network) config-json callback))
+
+(defn input-connection-string-for-importing-keypairs-keystores
+  "Provides connection string to status-go for the purpose of importing keypairs and keystores on the receiver side"
+  [connection-string config-json callback]
+  (log/info "[native-module] Sending Import Keypairs Connection String"
+            {:fn                :input-connection-string-for-importing-keypairs-keystores
+             :config-json       config-json
+             :connection-string connection-string})
+  (.inputConnectionStringForImportingKeypairsKeystores ^js (network)
+                                                       connection-string
+                                                       config-json
+                                                       callback))
```

Comment:
We may want to update this code to use the promise utility, which allows us to return a promise when calling these native functions instead of passing a callback directly. Here's an example of that: https://github.com/status-im/status-mobile/blob/d2fb6c7ee08857dc988b3914a74479d6e265cf87/src/native_module/core.cljs#L547-L561